website/docs/configuration/functions/templatefile.html.md

```diff
@@ -64,6 +64,59 @@ backend 10.0.0.2:8080
 
 ```
 
+### Generating JSON or YAML from a template
+
+If the string you want to generate will be in JSON or YAML syntax, it's
+often tricky and tedious to write a template that will generate valid JSON or
+YAML that will be interpreted correctly when using lots of individual
+interpolation sequences and directives.
+
+Instead, you can write a template that consists only of a single interpolated
+call to either [`jsonencode`](./jsonencode.html) or
+[`yamlencode`](./yamlencode.html), specifying the value to encode using
+[normal Terraform expression syntax](/docs/configuration/expressions.html)
+as in the following examples:
```

Comment:
What do you think of something like ending this sentence after the expression link and adding the following (see comment below for my thought process)
```suggestion
The following examples use a `for` expression ... 
```